tex-table/tex-table.rkt

```diff
@@ -190,14 +191,26 @@
     ("coprod" "∐")
     
     ("int" "∫")
+    ("iint" "∬")
+    ("iiint" "∭")
     ("oint" "∮")
+    ("oiint" "∯")
+    ("oiiint" "∰")
 
     ("sqrt" "√")
     
     ("skull" "☠") 
     ("smiley" "☺")
     ("blacksmiley" "☻")
     ("frownie" "☹")
+
+    ("Naturals" "ℕ")
```

Comment:
Should it be `Naturals` or `Natural`? If former, shouldn't `Complex` below be `Complexes` for consistency?